tests/functional/test_outputter.py

```diff
@@ -0,0 +1,47 @@
+# Aloe - Cucumber runner for Python based on Lettuce and Nose
+# Copyright (C) <2015> Danielle Madeley <danielle@madeley.id.au>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test level 3 outputter
+"""
+
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+# pylint:disable=redefined-builtin, unused-wildcard-import, wildcard-import
+from builtins import *
+# pylint:enable=redefined-builtin, unused-wildcard-import, wildcard-import
+from future import standard_library
+standard_library.install_aliases()
+
+from aloe.testing import (
+    FeatureTest,
+    in_directory,
+)
+
+
+@in_directory('tests/simple_app')
+class OutputterTest(FeatureTest):
+    """
+    Test level 3 outputter
+    """
+
+    def test_outputter(self):
+        """
+        Test Factory Boy steps.
+        """
+
+        self.run_features('features/highlighting.feature',
+                          verbosity=3)
```

Comment:
Some TODO is in order - I don't see that this tests anything other than not crashing.